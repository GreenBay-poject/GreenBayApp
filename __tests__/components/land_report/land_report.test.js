import React from 'react';
import renderer from 'react-test-renderer';
import Land_Report from '../../../components/land_report/land_report';

describe('<Land_Report />', () => {
    it('Sample Land_Report', () => {
        const tree = renderer.create(<Land_Report image={"https://apktada.com/storage/images/com/johartest/com.johartest_1.png"} setstep={() => { }} />).toJSON();
        console.log(tree)
        expect(tree).toMatchSnapshot();
        expect(tree.type).toBe('RCTScrollView');
        expect(tree.children[0].type).toBe("View");


    });
});