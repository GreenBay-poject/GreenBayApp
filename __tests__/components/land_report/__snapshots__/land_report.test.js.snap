// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Land_Report /> Sample Land_Report 1`] = `
<RCTScrollView>
  <View>
    <View>
      <View
        style={
          Object {
            "marginBottom": 0,
            "marginTop": 0,
          }
        }
      >
        <ActivityIndicator
          animating={true}
          color="#C00"
          hidesWhenStopped={true}
          size="large"
          style={
            Array [
              Object {
                "height": 80,
              },
            ]
          }
        />
      </View>
      <View
        style={
          Object {
            "marginBottom": 2,
            "marginTop": 2,
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "backgroundColor": "#153d37",
              "borderRadius": 10,
              "opacity": 1,
              "padding": 10,
              "width": "95%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "fontSize": 14,
              }
            }
          >
            BACK
          </Text>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
